core/src/test/kotlin/org/evomaster/core/search/gene/GeneTest.kt

```diff
@@ -205,7 +206,7 @@ class GeneTest {
         }
     }
 
-    //@Disabled("check seed 1616")
+    @Disabled
```

Comment:
so this is still failing?

Reply:
Unfortunately yes. The seed "1796" is failing. I debugged the issue and it seems to be a problem in a NumericStringGene that has a BigDecimalGene that isLocallyValid() is failing. It seems to be the same issue (the max value is excluded, but the big decimal's value is tha max value)

Reply:
It would be great if @ZhangMan1126 can have at look it.